tweet-stepengine/src/main/java/org/tweetwallfx/tweet/stepengine/dataprovider/TweetStreamDataProvider.java

```diff
@@ -93,16 +93,29 @@ private void updateImage(final Tweet tweet) {
                         sis -> latestTweetedImage = new Image(sis.get())));
     }
 
-    private void addTweet(final Tweet tweet) {
+
```

Comment:
nitpick: why the additional empty line?

Reply:
Bad formatting. Provider needs rewrite (not append, prepend as boolean but maybe enum). Will fix later.